llvm/include/llvm/CodeGen/SDPatternMatch.h

```diff
@@ -758,6 +778,40 @@ inline auto m_False() {
       m_Value()};
 }
 
+struct CondCode_match {
+  std::optional<ISD::CondCode> CCToMatch;
+  ISD::CondCode *BindCC;
+
+  explicit CondCode_match(ISD::CondCode CC) : CCToMatch(CC), BindCC(nullptr) {}
```

Comment:
Uses `ISD::CondCode *BindCC = nullptr` in the class body and you don't have to mention it here.

Reply:
Done.